care/emr/resources/questionnaire/utils.py

```
@@ -106,6 +106,32 @@ def validate_data(values, value_type, questionnaire_ref):  # noqa PLR0912
     return errors
 
 
+def normalize_value(value):  # noqa PLR0911
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)